### PR TITLE
BaSP-MR-10-58: Update Subscription schema and populate of member and class

### PR DESCRIPTION
**Pull request by enriqueanastasini**
Update Subscription schema {module} and populate of member and class {controller}

### CODE REVIEW

**Review by EmaBaldor — APPROVED** [review]

**Review by Flor-Farruggia — COMMENTED** [review]
i'm not sure, but maybe you gona need to put this info from models/Subscription.js like an array [ Schema.Types.ObjectId] but you should consult with the tutors because i'm not sure

**Comment by enriqueanastasini** [comment]
> i'm not sure, but maybe you gona need to put this info from models/Subscription.js like an array [ Schema.Types.ObjectId] but you should consult with the tutors because i'm not sure

I think that i asked about that in the meeting. Each subscription only has one class and one member so the array is not necesarily. Maybe I'm wrong or didn't fully understand. I will wait the comments of the tutors. thx

**Review by NicoCagnina — APPROVED** [review]

File: src/models/Subscription.js [hunk 1]
```diff
@@ -5,10 +5,12 @@ const { Schema } = mongoose;
 const subscriptionSchema = new Schema({
   classId: {
     type: Schema.Types.ObjectId,
+    ref: 'Class',
     required: true,
   },
   members: {
     type: Schema.Types.ObjectId,
```
Comment (NicoCagnina) [inline src/models/Subscription.js:12]: this should be an array

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/models/Subscription.js (hunk 1 above)
Comment (enriqueanastasini) [reply src/models/Subscription.js]: Done

File: src/models/Subscription.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ const { Schema } = mongoose;
 
 const subscriptionSchema = new Schema({
   classId: {
-    type: Schema.Types.ObjectId,
+    type: [Schema.Types.ObjectId],
```
Comment (nicolascagnina) [inline src/models/Subscription.js:7]: This is not an array. Just the members entity

**Review by nicolascagnina — CHANGES_REQUESTED** [review]

File: src/models/Subscription.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ const subscriptionSchema = new Schema({
     required: true,
   },
   members: {
-    type: Schema.Types.ObjectId,
+    type: [Schema.Types.ObjectId],
     required: true,
```
Comment (JuanMornik) [inline src/models/Subscription.js:12]: add a ref attribute please

File: src/models/Subscription.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ const subscriptionSchema = new Schema({
...
```
Comment (JuanMornik) [inline src/models/Subscription.js:8]: add a ref attribute please (may match with the name in populate)

**Review by JuanMornik — CHANGES_REQUESTED** [review]

**Review by JuanMornik — APPROVED** [review]

**Review by NicoCagnina — APPROVED** [review]

**Pull request merged**